src/neural/onnx/converter.cc

```diff
@@ -203,6 +230,10 @@ std::string Converter::MakeSqueezeAndExcite(
     const std::string& input, const std::string& name) {
   const int se_filters = se_unit.b1.size();
 
+  for (static bool flag = false; !flag; flag = true) {
```

Comment:
Is the intention to have common initializer for several blocks?

I think it won't work e.g. if network will be changed during the game (we support this), or if in selfplay player1 and player2 use different ONNX networks (that happens during match games).
Maybe add the (non-static) flag as the Converter member variable instead?